doc/release/release_dev.rst

```diff
@@ -39,6 +39,8 @@ Bugfixes
 Deprecations
 ------------
 
+- The function ``filters.lpi_filter.inverse`` was deprecated. Please use
+``filters.lpi_filter.filter_inverse`` instead.
```

Comment:
We don't normally recommend using functions from sub-submodules.

Reply:
If yes, it's done :slightly_smiling_face:

Reply:
My understanding would be to write something like:
```
- In ``filters``, the function ``lpi_filter.inverse`` was deprecated. Please use
``lpi_filter.filter_inverse`` instead.
```
or
```
- The function ``lpi_filter.inverse`` from ``filters`` was deprecated. Please use
``lpi_filter.filter_inverse`` instead.
```
But I'm not sure :upside_down_face:

Reply:
This is all internal implementation details, so I would recommend we say that `filters.inverse` was deprecated, please use `filters.filter_inverse`.